org.eclipse.dawnsci.remotedataset.server/src/org/eclipse/dawnsci/remotedataset/server/info/TreeServlet.java

```diff
@@ -92,17 +96,28 @@ private void doHandle(HttpServletRequest  request,
 		
 		try {
 			final ILoaderService lservice = ServiceHolder.getLoaderService();
+			lservice.clearSoftReferenceCache();
+			long startTime = System.currentTimeMillis();
+			
 			final IDataHolder holder = lservice.getData(path, true, new IMonitor.Stub());
-					        
 			final Tree tree = holder.getTree();
+			
+			long endTime = System.currentTimeMillis()-startTime;
+			
+			if (endTime > 100 && endTime < 500) {
```

Comment:
This 100 and 500 should be configurable for the server:
minEnd = Long.getLong("org.eclipse.dawnsci.remotedataset.minInterval", 100);
maxEnd = Long.getLong("org.eclipse.dawnsci.remotedataset.maxInterval", 500);

if (endTime > minEnd && endTime < maxEnd) {
...
}